<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>查看课程</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>591</width>
       <height>321</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>课程号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>课程名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学分</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>授课教师</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>地点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>选课人数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>修改</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>删除</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addCourse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加课程到选课列表中</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>管理用户信息</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>修改密码</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>输入新密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>再次输入新密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirmPassword">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="changePassword">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>输入原密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newPassword">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prevPassword">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
